docs/lessons/cherry-pickr.md

```diff
@@ -124,9 +124,9 @@ Now we are able to move our player! In the simulator, test out the movement of t
 
 We want cherries to appear randomly on the screen every half a second. In order to do this, go into ``||game:Game||`` in the Toolbox and drag out the ``||game:on game update every||`` block onto the editor.  
 
-Next we will define what a cherry is by going to ``||sprites:Sprites||`` and dragging out the ``||sprites:set agent to||`` block inside the ``||game:on game update every||`` block.  
+Next we will define what a cherry is by going to ``||sprites:Sprites||`` and dragging out the ``||sprites:set mySprite to||`` block inside the ``||game:on game update every||`` block.  
 
-Click the drop down and change ``agent`` to ``item``. Click the gray square box to draw what you want your item to look like. Then click the drop down that says ``Player`` and click ``Add a new kind...``. Name your kind ``Item``.
+Click the drop down and rename ``mySprite`` to ``item``. Click the gray square box to draw what you want your item to look like. Then click the drop down that says ``Player`` and click ``Add a new kind...``. Name your kind ``Item``.
```

Comment:
Since this is the second sprite that is created, it should be ``mySprite2`` here (should be easier to clarify by just directing them to change the name of the sprite they just dragged out, instead, in case they've dragged out other ones as well).

Reply:
and maybe we should also change the variable name from ``item`` to ``cherry``?

Reply:
Good catch. New commit arriving shortly.

I went back-and-forth in my mind about changing the `item` variable to `cherry`. To me, `item` flows better in the text. I can change it, though, if you feel strongly about it.

Reply:
I wouldn't call it a strong feeling, I just tend to lump `item` into the category of names like `list`, `iter`, `num`, etc., where they don't really say anything about what's stored. Doesn't really matter in this case at all, I just tend to rename stuff like that pretty aggressively because naming things 'well' can be hard to learn (at least in the cs intro courses)

Reply:
In principle, I certainly agree. I teach the same thing in my classes.

We're using the default names for just about everything in this lesson, though. However, since we're already renaming it from `mySprite2`, we probably could use a name that's more meaningful.

I'll work on it.

Reply:
OK; I forgot that I had removed some of the narrative that involved `item` anyway. The remaining phrasings are a lot less awkward than I thought they would be. New commit arriving shortly.